main.go

```diff
@@ -258,6 +267,13 @@ func loadConfig(filename string, queryUserData bool) (*Config, error) {
 		UsersDir:                   "C:\\Users",
 		CleanUpTaskDirs:            true,
 		IdleShutdownTimeoutSecs:    0,
+		WorkerTypeMetaData: map[string]interface{}{
```

Comment:
instead of creating something generic here with string to an empty interface, why not just create a type for this data?